### PR TITLE
Upgrade `cli-boxes` dependency

### LINKED ISSUE #68: Upgrade `cli-boxes` dependency
https://github.com/sindresorhus/cli-boxes/releases/tag/v3.0.0

### PR DESCRIPTION
**Pull request by Caesarovich**
Fixes #68 

I updated the dependency, and modified all documentation according to the new options provided by cli-boxes v3.
I also took the time to refactor the code responsible for 'inboxing/incasing' the content into a separate function for better readability and maintainability. The tests and snapshots also have been updated.

Let me know about any desired changes !

### CODE REVIEW

**Comment by Caesarovich** [comment]
Forgot to @sindresorhus my bad.

**Comment by sindresorhus** [comment]
> Forgot to @sindresorhus my bad.

No need to ;) Maintainers/owners of a repo get notified of new pull requests.

File: readme.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ Color of the box border.
 Type: `string | object`\
 Default: `'single'`\
 Values:
+- `'arrow'`
+```
+↘↓↓↓↙
+→foo←
+↗↑↑↑↖
+```
```
Comment (sindresorhus) [inline readme.md:77]: This should be last.

File: tests/border-option.js [hunk 2]
```diff
@@ -112,8 +112,10 @@ test('border style (custom ascii style)', t => {
 			topRight: '2',
 			bottomLeft: '3',
 			bottomRight: '4',
-			horizontal: '-',
-			vertical: '|',
+			top: '-',
+			bottom: '_',
+			left: '|',
+			right: '!',
```
Comment (sindresorhus) [inline tests/border-option.js:118]: Typo

**Comment by sindresorhus** [comment]
Can you preserve support for using `horizontal` and `vertical`? You can just normalize the option right away. Not worth breaking people's use of this.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: tests/border-option.js (hunk 2 above)
Comment (Caesarovich) [reply tests/border-option.js:118]: I'm not sure what you meant there, I swapped top/bottom with left/right in the next push.

Comment (sindresorhus) [reply tests/border-option.js:116]: Did you mean to use an exclamation mark instead of a `|`?

Comment (Caesarovich) [reply tests/border-option.js:116]: No I meant to use 4 different characters.

**Comment by Caesarovich** [comment]
Did we have anything else to change ?

**Pull request merged**